src/clj/ataru/tarjonta_service/tarjonta_service.clj

```diff
@@ -32,7 +32,7 @@
 
 (defn- forms-in-use
   [cache-service organization-service session]
-  (let [direct-organizations    (select-organizations-for-rights session [:form-edit])
+  (let [direct-organizations    (select-organizations-for-rights organization-service session [:form-edit])
```

Comment:
`select-organizations-for-rights` muutoksen jälkeen täällä ei ole enää tarpeen tehdä `organization-protocol/get-all-organizations` kutsua rivillä 39.